### ISSUE TITLE
Can't use localsConvention in cssLoaderOptions

### ISSUE THREAD

**Issue by NeatNit** [vercel/next-plugins#546]
My next.config.js file looks like this:

```javascript
const withCSS = require('@zeit/next-css');
const path = require('path');

const cssModuleRegex = /\.module\.css$/;

module.exports = withCSS({
  cssModules: true,
  cssLoaderOptions: {
    localsConvention: 'dashes',
    getLocalIdent: (loaderContext, _, localName) => {
      const fileName = path.basename(loaderContext.resourcePath);

      if (cssModuleRegex.test(fileName)) {
        const name = fileName.replace(/\.[^/.]+$/, '');
        return `${name}__${localName}`;
      }

      return localName;
    },
  },
});
```

The `getLocalIdent` bit works fine, but for some reason `localsConvention` is ignored. My imported styles still don't have camelCase names - e.g. `.some-class` can only be accessed by `importedStyle['some-class']`, not `importedStyle.someClass`.

I can't for the life of me figure out why, am I doing something wrong?

**Comment by Ben07** [comment]
have you resolve the problem?

**Comment by sever1an** [comment]
The documentation for `css-loader` options points to the master branch, but the plugin imports `^2.1.0`. 

The `localsConvention` option is actually *very* new, previously it was `camelCase` with slightly different values.

Here is the correct reference: https://github.com/webpack-contrib/css-loader/blob/v2.1.0/README.md#camelcase

TL;DR - `{ camelCase: 'dashes' }`

**Comment by timneutkens** [comment]
Hi, thanks for creating an issue. We currently recommend using https://nextjs.org/docs/basic-features/built-in-css-support as the plugins have been deprecated in favor of the built-in support.
